packages/app/ui/navigation/dashboardNavigation.tsx

```diff
@@ -16,10 +15,7 @@ export function DashboardNavigation({
   return (
     <View className="flex flex-1 max-h-[100vh] bg-blue-dark overflow-hidden">
       <Navbar />
-      <View className="flex flex-row flex-1">
-        {tw.prefixMatch("sm") && <DashboardTabBar column />}
-        {children}
-      </View>
+      {children}
```

Comment:
@alejomendoza, I removed dashboard tabbar from desktop view in favour of navbar links. What do you think about it, should I put it back and hide navbar links or should I be showing both.

Reply:
Good question. Dashboard tabbar should be in place when logged in, navbar links only show up on logged out. 
Looking great awesome job.